convert2rhel/backup/packages.py

```diff
@@ -134,37 +134,20 @@ def enable(self):
             if system_info.eus_system and system_info.id == "centos":
                 self.reposdir = get_hardcoded_repofiles_dir()
 
-            if not system_info.has_internet_access:
-                if self.reposdir:
-                    loggerinst.debug(
-                        "Not using repository files stored in %s due to the absence of internet access." % self.reposdir
-                    )
-
-                for pkg in self.pkgs:
-                    self._backedup_pkgs_paths.append(
-                        utils.download_pkg(
-                            pkg=pkg,
-                            dest=BACKUP_DIR,
-                            set_releasever=self.set_releasever,
-                            custom_releasever=self.custom_releasever,
-                            varsdir=self.varsdir,
-                        )
-                    )
```

Comment:
Do we need to do this? In either case, it's being handled in #1181 so whichever gets merged first will have to fix

Reply:
@bocekm said that we need to use repository files regardless if there is internet or not, so I removed the internet access dependency here

Reply:
That's fine, either one that get merged first, we can deal with the rebase in the other PR.